### ISSUE TITLE
Gun decay / keys 

### ISSUE THREAD

**Issue by skyy-reginald** [InZidiuZ/op-framework-issue-tracker#2317]
**Summary:**

When bringing guns into the city the time it takes to get enough for a shipment and then push out to the mass takes a little bit of time. During that time the guy na have already decayed anywhere from 15%-30%. 

If it’s pushed to a middle man and to en to the mass it is then down even further on percentage for decay. 

I would like to suggest a week to 2 week period where storing in a box in a house won’t decay the gun just so they masses can get a brand new gun and not a half dead one but while during that 2 week period if they use the gun it still decays on use as normal. After the 1-2 week period it would then decay on time and use. 

Keys

Keys are very hard to come by when now makes it hard to bring guns in. I would like to suggest another way to get keys for guns. Possibly chop or garbage or something. Just so it’s not so hard to get the keys. Right now it can be really hard to get orders done for people due to the lack of keys. Keys also have a decently fast decay time so you want need to worry about people holding on to them for long 

**Reason:**
I believe this would be a good change to help with the gun market so that the masses get fresh guns and not half dead ones. 

The key issue can help wi to jewelry stores and not only guns as well and can create a little more rp by the people having to run guns to buy from said people.

**Comment by MetaHatter** [comment]
A gun at 80-90% health is not "half dead" it's still VERY usable for an extended period of time.  For some reason people just want to see a full white bar on purchase. IMHO People just need to learn that no gun will be pristine, ever.

I do not think it's necessary as the current decay adds urgency to the supplier and middle-man to move the product as quickly as possible.  If we stop decay for the suppliers, the middle-men would be in here the next day asking we delay it for them too. (Guaranteed)

That being said, if this WAS to be done, I would suggest keeping the 10 week decay period, just not start it until after week 1 or 2.

**Comment by skyy-reginald** [comment]
I don’t think you fully understood what I said.  I’m not asking to get rid of decay but to delay it for a week or 2. 

I am also not saying they get it at 80-90%. Once the middle man sells the masses get it at anywhere between 70-50% so yes half dead is a correct term to use. 

1 week is enough for the suppliers and 1 week is enough for the middle man. Anything more that that isn’t needed. 

Right now there is a time limit on running them so it take all day of non stop running. So as much as you preached urgency it’s hard to be urgent with a timer on it as well. 

All I’m asking in a delay to give suppliers and middle man time to give guns to the mass at better percentage than half dead. 

This not only lets the masses of criminals have better guns for longer. It gives cops time to find the guns and raid for them as well. Then the cops may find better guns and not dead ones by the time we can get everything approved to make a raid happen.

**Comment by twooot** [comment]
Lets take the advancedrifle for example. It decays fully after 75 days (that is 2 and a half months). Now lets say its durability is at 70%, that would mean it still has 3,150 shots left before it degrades or if you dont use it it would still be usable for 52 days (more than 1 and a half months).

60-70% is not half dead, it is very much still usable for extended periods of time.

TL;DR: No :)
